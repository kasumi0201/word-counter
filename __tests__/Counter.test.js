import React from 'react';
import Counter from "../src/Counter";
import renderer from "react-test-renderer";

describe ("the count function",()=>{
  it ("displays the count",()=>{
    const counter = renderer.create(<Counter legend="Count" count={34}/>)
    expect(counter.toJSON).toMatchSnapshot();
  });
});
