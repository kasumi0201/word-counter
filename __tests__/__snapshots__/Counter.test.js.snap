// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the count function displays the count 1`] = `[Function]`;
